translations/monero-core_de.ts

```diff
@@ -2770,7 +2775,7 @@ Spending address index: </source>
     <message>
         <location filename="../main.qml" line="887"/>
         <source>Balance (syncing)</source>
-        <translation>Guthaben (synchronisierend)</translation>
+        <translation>Guthaben (synchronisiere..)</translation>
```

Comment:
Change this back please, we had a long discussion about it already with some translator.

Reply:
Although 'synchronisierend' is the "correct" translation and I will change it, I also want to throw in that 'synchronisierend' sounds really elitist. Nobody talks like that imho.